api/services/oidc/knexAdapter.js

```diff
@@ -0,0 +1,91 @@
+const oidc_payloads = "oidc_payloads"
+
+const types = [
+  "Session",
+  "AccessToken",
+  "AuthorizationCode",
+  "RefreshToken",
+  "DeviceCode",
+  "ClientCredentials",
+  "Client",
+  "InitialAccessToken",
+  "RegistrationAccessToken",
+  "Interaction",
+  "ReplayDetection",
+  "PushedAuthorizationRequest",
+  "Grant",
+].reduce((map, name, i) => ({ ...map, [name]: i + 1 }), {})
```

Comment:
This feels a wee bit fragile given that if another type is not carefully added (at the end) later then the mapping could be shifted and I presume break things in a bad way. Maybe a comment and/or just a more explicit mapping?